lib/parse.js

```diff
@@ -839,6 +839,7 @@ const parse = (input, options) => {
 
         state.output += prior.output + prev.output;
         consume(value + advance());
+        value = '/';
 
         push({ type: 'slash', value, output: '' });
```

Comment:
Good catch! 

Instead of reassigning `value`, let's move that to the token:

```js
push({ type: 'slash', value: '/', output: '' });
```

Reply:
JFYI: I use reassigning `value` only for local consistency within this module (in most cases we use `{ value }` without property value assigning).

Reply:
Thanks for pointing that out, and I appreciate you paying attention to those details!